packages/amplify_auth_cognito/lib/src/CognitoSession/AWSCognitoUserPoolTokens.dart

```diff
@@ -22,11 +22,9 @@ class AWSCognitoUserPoolTokens {
   String accessToken;
   String idToken;
   String refreshToken;
-  AWSCognitoUserPoolTokens.init({@required LinkedHashMap<dynamic, dynamic> tokens}) {
-    if (tokens != null) {
-      this.accessToken = tokens["accessToken"];
-      this.idToken = tokens["idToken"];
-      this.refreshToken = tokens["refreshToken"];
-    }
-  }
+  AWSCognitoUserPoolTokens.init(
+      {required LinkedHashMap<dynamic, dynamic> tokens})
+      : this.accessToken = tokens["accessToken"],
```

Comment:
This is a scenario in which null safety ideally should force you to do a null check, but it looks like it doesn't force you to do that with dynamics. It would if the type of tokens was `LinkedHashMap<String, String>`, not `LinkedHashMap<dynamic, dynamic>`. I'm not saying we should necessarily change it. `LinkedHashMap<dynamic, dynamic>` might be the correct type. Its definitely something to pay attention to though.

Reply:
The method throwing the error is Amplify.Auth.fetchAuthSession(). I could only repro this on iOS

Reply:
Working on a fix for this - for now on the platform side.

Reply:
I also think <String, String> is right here - going to play around with it.

Reply:
Hmm - I think we had this as dynamic because of how the data was typed coming over the method channel.